modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -47,6 +51,13 @@ def verify_power_of_attorney
           verifier.verify(header('X-Consumer-PoA'))
         end
       end
+
+      def use_headers?
+        # if any of the required headers are present we should attempt to use headers
+        request.headers['X-VA-SSN'].present? ||
```

Comment:
you could probably clean this up with an array of headers, and a reduce or something like that

Reply:
updated the method